definitions/private-law/json/CaseEvent.json

```diff
@@ -89,5 +89,31 @@
     "SecurityClassification": "Public",
     "ShowSummary": "Y",
     "ShowEventNotes": "N"
+  },
+  {
+    "LiveFrom": "05/11/2021",
+    "CaseTypeID": "C100",
+    "ID": "applicantsDetails",
+    "Name": "Applicant details",
+    "Description": "Details of the applicants in the case",
+    "DisplayOrder": 88,
+    "SecurityClassification": "Public",
+    "PreConditionState(s)": "AWAITING_SUBMISSION_TO_HMCTS",
+    "PostConditionState": "AWAITING_SUBMISSION_TO_HMCTS",
+    "ShowSummary": "Y",
+    "ShowEventNotes": "N"
+  },
+  {
+    "LiveFrom": "05/11/2021",
+    "CaseTypeID": "C100",
+    "ID": "childDetails",
+    "Name": "Child details",
+    "Description": "Details of the children in the case",
+    "DisplayOrder": 89,
```

Comment:
This should be 9

Reply:
Once all events are done, we will reorder this. it was intentionally added with higher number